Watchdog/Watchdog.ino

```diff
@@ -27,34 +27,48 @@ void loop() {
     
     //NOT STARTED YET
     case 0:
+      currTime = millis();
       //check for change in hearbeat
       heartBeat = digitalRead(inPin);
       if (heartBeat != lastBeat){ changeCounter++; }
-      totalCounter++;
+      
       lastBeat = heartBeat;
     
-      //if we get 10 changes in a span of 15 sec continue to healthy state
+      //if we get 5 changes in a span of 260ms continue to healthy state
       if (changeCounter > goal){
+        //sets everything up for the next state
+        changeCounter = 0;
```

Comment:
Create a function `setupState1()` that sets all of the variables to their correct values.